src/test/java/no/unit/nva/doi/transformer/utils/DataciteTypesUtilTest.java

```diff
@@ -0,0 +1,99 @@
+package no.unit.nva.doi.transformer.utils;
+
+import no.unit.nva.doi.transformer.model.internal.external.DataciteResponse;
+import no.unit.nva.doi.transformer.model.internal.external.DataciteTypes;
+import no.unit.nva.model.PublicationType;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class DataciteTypesUtilTest {
+
+    public static final String DATASET = "Dataset";
+
+    @DisplayName("The mapper util checks for resourceTypeGeneral and returns null for non-text values")
+    @Test
+    void testMapperReturnsNullWhenInputIsNotTypeText() {
+        DataciteTypesUtil.mapToType(getNonTextDataciteResource());
+    }
+
+    @DisplayName("The mapping utility returns type JOURNAL_CONTENT when all types agree")
```

Comment:
What does it mean that "two types agree" ? Does it mean that they are Journal Content?